rllib/utils/numpy.py

```diff
@@ -559,3 +560,22 @@ def softmax(
     # return x_exp /
     #   np.maximum(np.sum(x_exp, axis, keepdims=True), SMALL_NUMBER)
     return np.maximum(x_exp / np.sum(x_exp, axis, keepdims=True), epsilon)
+
+
+@PublicAPI
+def zeros_like(object: Union[np.ndarray, Dict, List, Tuple]):
```

Comment:
hey actually you may be able to do this with simply:
```
tree.map_structure(np.zeros_like, object)
```
give it a try, would save you some work. also probably shouldn't use object as the parameter name.

Reply:
Loved the idea. So cool man. you are a tree master :) I gotta learn this library better.